modal/file_io.py

```diff
@@ -166,6 +184,53 @@ async def _consume_output(self, exec_id: str) -> AsyncIterator[Optional[bytes]]:
             for message in batch.output:
                 yield message
 
+    async def _consume_watch_output(self, exec_id: str) -> None:
```

Comment:
We've got identical consuming code to this in a few places. Cant it be genericized into an iterator where the only thing that changes is the work you're doing in the body?